index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <div class="nav-bar">
+            <div class="nav-logo">
+                <h1>My Logo</h1>
+            </div>
+            <nav class="nav-links">
+                <div class="ambuger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main class="main-card">
+        <section>
+            <div class="main-container">
+                <div class="intro-text">
+                    <h2>I'm Emmanuel Glad to see you!</h2>
+                    <p>
+                        I am a Software developer, i can help you build a product, feature
+                        or website, look through some of my works and experience! if you
+                        like what you see and have a project you need coded dont hesitate
+                        to contact me.
+                    </p>
+                </div>
+                <div class="intro-items">
+                    <a href="#">LET'S CONNECT</a>
```

Comment:
- [ ] It would be better also to replicate the font and colors (background and font) defined for the template by removing the default generated underlined text-decoration and replicating the exact font 😃 

## Your design
![image](https://user-images.githubusercontent.com/102757126/211802416-01dff076-63ff-4c6d-8eb1-a17c867d4ef0.png)

## Figma design
![image](https://user-images.githubusercontent.com/102757126/211802914-70695f74-094d-4919-a3ac-63937d30e5e9.png)